tests/test_load_adrs.py

```diff
@@ -0,0 +1,347 @@
+import re
+import subprocess
+from mock import patch
+from datetime import datetime
+from decimal import Decimal
+
+import pytest
+import manage
+from webservices import rest
+from webservices.legal_docs import DOCS_INDEX
+from webservices.legal_docs.utils import parse_statutory_citations
+from tests.common import TEST_CONN, BaseTestCase
+
+@pytest.mark.parametrize("test_input,case_id,entity_id,expected", [
+    ("431", 1, 2,    # With reclassification
+        [{'text': '431',
+          'title': '2',
+          'type': 'statute',
+          'url': 'https://api.fdsys.gov/link?collection=uscode&year=mostrecent&link-type=html'
+          '&title=52&section=30101'}]),
+    ("30116", 1, 2,  # Already reclassified
+        [{'text': '30116',
+          'title': '52',
+          'type': 'statute',
+        'url': 'https://api.fdsys.gov/link?collection=uscode&year=mostrecent&link-type=html&title=52&section=30116'}]),
+    ("434(a)(11)", 1, 2,
+        [{'text': '434(a)(11)',
+          'title': '2',
+          'type': 'statute',
+        'url': 'https://api.fdsys.gov/link?collection=uscode&year=mostrecent&link-type=html&title=52&section=30104'}]),
+    ("9999", 1, 2,
+        [{'text': '9999',
+          'title': '2',
+          'type': 'statute',
+        'url': 'https://api.fdsys.gov/link?collection=uscode&year=mostrecent&link-type=html&title=2&section=9999'}]),
+    ("9993(c)(2)", 1, 2,
+        [{'text': '9993(c)(2)',
+          'title': '2',
+          'type': 'statute',
+        'url': 'https://api.fdsys.gov/link?collection=uscode&year=mostrecent&link-type=html&title=2&section=9993'}]),
+    ("9993(a)(4) formerly 438(a)(4)", 1, 2,
+        [{'text': '9993(a)(4)',
+          'title': '2',
+          'type': 'statute',
+        'url': 'https://api.fdsys.gov/link?collection=uscode&year=mostrecent&link-type=html&title=2&section=9993'}]),
+    ("9116(a)(2)(A), 9114(b) (formerly 441a(a)(2)(A), 434(b)), 30116(f) (formerly 441a(f))", 1, 2,
+        [{'text': '9116(a)(2)(A)',
+          'title': '2',
+          'type': 'statute',
+        'url': 'https://api.fdsys.gov/link?collection=uscode&year=mostrecent&link-type=html&title=2&section=9116'},
+        {'text': '9114(b)',
+          'title': '2',
+          'type': 'statute',
+        'url': 'https://api.fdsys.gov/link?collection=uscode&year=mostrecent&link-type=html&title=2&section=9114'},
+        {'text': '30116(f)',
+          'title': '52',
+          'type': 'statute',
+        'url': 'https://api.fdsys.gov/link?collection=uscode&year=mostrecent&link-type=html&title=52&section=30116'}]),
+    ("9993(a)(4) (formerly 438(a)(4)", 1, 2,  # No matching ')' for (formerly
+        [{'text': '9993(a)(4)',
+          'title': '2',
+          'type': 'statute',
+        'url': 'https://api.fdsys.gov/link?collection=uscode&year=mostrecent&link-type=html&title=2&section=9993'}]),
+])
+
+def test_parse_statutory_citations(test_input, case_id, entity_id, expected):
```

Comment:
My point was that if the underlying function is used by both ADRs and MURs, one test would suffice; it does not have to be repeated for both ADRs and MURs.